modules/integration/src/test/java/org/apache/fluo/integration/impl/OracleIT.java

```diff
@@ -226,7 +226,7 @@ public void singleOracle_goesAwayAndComesBack() throws Exception {
     sleepWhileConnected(oserver);
 
     int count = 0;
-    while (count < 5 && client.getOracle() != null) {
+    while (count < 20 && client.getOracle() != null) {
```

Comment:
Rather than retry a certain number of times, maybe it would be better to retry indefinitely, and add a TimeoutRule to the JUnit test instead.